guidelines to avoid impersonation attacks (#117)
thomas-fossati/draft-psa-token: draft-tschofenig-rats-psa-token.md

Note, however, that the use of MAC authentication is NOT RECOMMENDED due to the associated
infrastructure costs for key management and protocol complexities.

> A PSA attester MUST NOT expose the PSA attestation API to unauthenticated callers.

Author: ```suggestion
```

Note, however, that the use of MAC authentication is NOT RECOMMENDED due to the associated
infrastructure costs for key management and protocol complexities.

> A PSA attester MUST NOT provide attestation evidence to an unauthorized challenger.

Reviewer: I have a concern about `unauthorized`, the only alternative I can think of is `untrusted`.

Author: done in [a0b78aa](https://github.com/thomas-fossati/draft-psa-token/pull/117/commits/a0b78aa9537e2f0ff2c087cbce87aa0828e18928)

> A PSA attester MUST NOT provide attestation evidence to an unauthenticated challenger.

Author: ```suggestion
A PSA attester MUST NOT provide attestation evidence to an unauthorized challenger.
```

> Additionally, a PSA attester MUST NOT send attestation evidence to an unauthenticated challenger.

Author: ```suggestion
A PSA attester MUST NOT provide attestation evidence to an unauthenticated challenger.
```

> Failing to do so may may allow an attacker to interpose between the attester and a verifier, tricking the verifier into believing the attacker is the legitimate attester.

Reviewer: This line is very long

Reviewer: Sounds good to me

Author: done in [a0b78aa](https://github.com/thomas-fossati/draft-psa-token/pull/117/commits/a0b78aa9537e2f0ff2c087cbce87aa0828e18928)
